src/components/gds-card/gds-card.stories.js

```diff
@@ -0,0 +1,21 @@
+import { html } from 'lit-html'
+
+export default {
+  title: 'Containers',
+  component: 'gds-card',
+}
+
+// prettier-ignore
+export const Card = () => html`
+  <p>
+    <gds-card>
+      A card that renders content in a container that has * white background and border shadows. Comes without
+      padding by default.
+    </gds-card>
+  </p>
+  <p>
+    <gds-card style="padding: 40px;">
```

Comment:
Is this how the card will be used in production as well? Isn't padding such a common use case for cards, that it should be built-in to the card?

A typical option would be to have multiple named slots in the card, where e.g. the content part has padding but the thumbnail part doesn't, and if you want to use custom padding or no padding you just don't use any of those slots.

Another option would be to allow specifying either padding true|false, or padding small|medium|large. That's very limited though, since you typically have different padding for different parts of the card.

Or will there be type-specific card implementations that take care of the padding, like `gds-person-card`?

Main point is that we should never have to use "40px" in code, but instead we could rely on the card using a standard `$gds-spacing-s` padding (or whatever we call it) every time.

Reply:
I guess `gds-media-card` answers my question now that I read that far :)

Reply:
Yeah, the idea there was just to show that you can (if you want) use normal styles to style this web component. 

I agree that we should definitely have a structured approach to spacing, but that design meeting with Antti has been pushed back about 4 times now in my calendar. @henritukiainen, it would be good it you would also participate in that meeting.